numismatic/feeds/base.py

```diff
@@ -49,6 +48,10 @@ class Subscription:
     def market_name(self):
         return f'{self.exchange}--{self.symbol}--{self.channel}'
 
+    async def start(self):
+        logger.info(f'Starting Subscription {self.market_name!r} ...')
+        await self.client._subscribe(self)
+
```

Comment:
Allows subscription to be started as soon as it is created with `asyncio.ensure_future()`.